pkg/ui/console/client.go

```diff
@@ -365,7 +365,7 @@ func (c Client) RunPolicies(ctx context.Context, policySource, outputDir string,
 		OutputDir:   outputDir,
 		RunCallback: policyRunCallback,
 	})
-	if resp != nil {
+	if resp == nil {
```

Comment:
I think these were fixed and rebase will conflict.

Reply:
ill fix the conflict on the rebase.